config/locales/modules/child-development-and-the-eyfs.yml

```diff
@@ -598,14 +617,10 @@ en:
             <p class="govuk-body">Do you know which areas of your environment are most used?  </p>
             <p class="govuk-body">Are there areas the children do not use?  </p>
             <p class="govuk-body">Does the environment strike the right balance between social interaction and quiet areas? </p>
-            <p class="govuk-body">How do practitioners in your setting make sure children hear and experience a range of conversation, including new and repeated vocabulary? </p>
-            <p class="govuk-body">How do you create opportunities to extend children's language?</p>
-            <p class="govuk-body">Are children who need help developing their communication skills receiving targeted, adult led interactions to help them develop their language? </p>
-            <p class="govuk-body">What could you do differently?  </p>
-
           </div>
         notes: true
 
+          It is important that you plan and regularly evaluate your learning environments to ensure that children remain curious and motivated to learn. You should also make sure there are opportunities for children to effectively learn, rehearse and develop their communication skills. 
```

Comment:
Which page should this sit on and where?

Reply:
I think this one will be fixed when the new reflection points are merged in so removing for now